views/options/book-this.erb

```diff
@@ -4,11 +4,14 @@
 
   <form method="post" action="/booking">
     <label for="pickup-location">Pickup location</label>
-    <select id="pickup-location" name="pickup-location" autocomplete="off">
-      <% option.pickup_locations.each do |pickup_location| %>
-        <option value="<%=pickup_location.code%>"><%=pickup_location.display%></option>
-      <% end %>
-    </select>
+    <div class="dropdown">
+      <select id="pickup-location" name="pickup-location" autocomplete="off">
+        <% option.pickup_locations.each do |pickup_location| %>
+          <option value="<%=pickup_location.code%>"><%=pickup_location.display%></option>
+        <% end %>
+      </select>
+      <%= erb :"components/dropdown_icon" %> 
+    </div>
```

Comment:
Assuming the question about dropdowns is how do we keep having to repeat this kind of thing, we could definitely do a dropdown partial that accounts for the usual needs of a dropdown. It takes an object that has a working each function, and what should be the value and what should be the display, what should be the name and what should be the id.

But also I think this is fine, and we can refactor when we get to actually have more options. 

Am I on the right track?

Reply:
Yes! A partial is a great idea.